wp-http-blocklist.php

```diff
@@ -56,7 +56,8 @@ function pre_http_request( $flag, $parsed_args, $url ) {
 
 	foreach ( $blocklist as $blocklist_domain ) {
 		if ( $request_host === $blocklist_domain ) {
-			$response = new \WP_Error( 'http_request_blocked', __( 'This URL is blocked from a deny list.', 'wp-http-blocklist' ) );
+			// translators: First is the host blocked, second is the full url called
+			$response = new \WP_Error( 'http_request_blocked', sprintf( __( 'Host %1$s with URL %2$s is blocked from a deny list.', 'wp-http-blocklist' ), $request_host, $url ) );
```

Comment:
@Rahe I'm not sure if the URL is useful if we already include the host in the error message.

Should we clean the URL before adding it to the error message (remove all the params etc.) ?

Reply:
Just the host sufficient you think in this case ?

Reply:
Yes, IMHO it's enough for debugging purposes.